docs/network-iot/lorawan-roaming.mdx

```diff
@@ -137,14 +137,11 @@ Roaming Console.
 
 ## Advanced Roaming
 
-### Session Key Filter
+There are additional configurations, automations, and data you can access with OpenLNS. You can
+learn more about these in the [Advanced Configuration](/iot/run-an-lns/advanced-configuration)
+section.
 
-The session key filter gives an opertator the ability to accept and reject specific roaming traffic
-on a device level. Leveraging the session key filter is recommended for all operators. Read more
-about the session key filter in the
-[LNS advanced configuration](/iot/run-an-lns/advanced-configuration#session-key-filter).
-
-### Proxy Service
+## Roamimg Proxy Service
 
 If Helium's OpenLNS Architecture is a deal breaker for you, then you can leverage
 [this service we prototyped](https://github.com/helium/helium-packet-router-ingest), which should
```

Comment:
Should probably rewrite this while we're in here

Reply:
@jthiller edited in 2nd commit